src/main/java/com/mwt/explorers/EpsilonGreedyExplorer.java

```diff
@@ -45,22 +45,23 @@ public void updatePolicy(Policy<T> newPolicy) {
     this.defaultPolicy = newPolicy;
   }
 
-  public ChosenAction chooseAction(long saltedSeed, T context) {
+  public DecisionTuple chooseAction(long saltedSeed, T context) {
+    int numActionsForContext = getNumActions(context);
     Random random = new Random(saltedSeed);
```

Comment:
@lhoang29 are we going to use the built-in C# Random generator as well or re-implement the current C version?
If we want blackbox test compat, we might all have to settle for the same random generator